src/Util/Tokens.php

```diff
@@ -629,6 +635,7 @@ final class Tokens
         T_ANON_CLASS => T_ANON_CLASS,
         T_INTERFACE  => T_INTERFACE,
         T_TRAIT      => T_TRAIT,
+        T_ENUM       => T_ENUM,
```

Comment:
I agree with this change in principle, but this **_will_** affect all sniffs looking for the `$ooScopeTokens` tokens and as the type declaration in the enum declaration is unique, I suspect that a number of sniffs referring to this token array will need updating to handle this correctly.

Reply:
Yes, but I think it should be better to solve it in other PRs. This PR should be merged first.